examples/cactus-example-supply-chain-backend/src/main/typescript/supply-chain-app.ts

```diff
@@ -200,6 +235,57 @@ export class SupplyChainApp {
     registryB.add(quorumConnector);
 
     await this.startNode(httpApiB, httpGuiB, registryB);
+
+    this.log.info(`Configuring Cactus Node for Ledger B...`);
```

Comment:
@jordigiam Super minor nit pick, apologies: I'm guessing that was going to be `Ledger C` not `B`

Reply:
Done